unyt/tests/test_unyt_array.py

```diff
@@ -2161,17 +2161,21 @@ def test_ones_and_zeros_like():
 
 
 def test_coerce_iterable():
-    from unyt import cm, km
+    from unyt import cm, m, g
 
     a = unyt_array([1, 2, 3], "cm")
-    b = [1 * cm, 2 * km, 3 * cm]
+    b = [1 * cm, 2 * m, 3 * cm]
```

Comment:
Seems a tiny bit more fragile (in a good way) like this
```suggestion
    b = [1 * cm, 2 * m, 3 * km]
```

Reply:
Your suggested change makes zero difference to the actual thing we’re testing, no.